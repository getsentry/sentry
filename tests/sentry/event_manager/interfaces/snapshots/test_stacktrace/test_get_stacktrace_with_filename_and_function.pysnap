---
created: '2019-03-14T17:12:36.632510Z'
creator: sentry
source: tests/sentry/event_manager/interfaces/test_stacktrace.py
---
errors: null
get_stacktrace: "Stacktrace (most recent call last):\n\n  File \"foo\", in biz\n \
  \ File \"bar\", in baz"
to_json:
  frames:
  - abs_path: foo
    filename: foo
    function: biz
  - abs_path: bar
    filename: bar
    function: baz
to_string: "Stacktrace (most recent call last):\n\n  File \"foo\", in biz\n  File\
  \ \"bar\", in baz"
