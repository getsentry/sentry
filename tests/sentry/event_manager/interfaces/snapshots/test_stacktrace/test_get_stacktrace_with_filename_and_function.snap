errors: null
get_stacktrace: "Stacktrace (most recent call last):\n\n  File \"foo\", in biz\n \
  \ File \"bar\", in baz"
to_json:
  frames:
  - {abs_path: foo, filename: foo, function: biz}
  - {abs_path: bar, filename: bar, function: baz}
to_string: "Stacktrace (most recent call last):\n\n  File \"foo\", in biz\n  File\
  \ \"bar\", in baz"
